content/ur/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "کلاؤڈ کی مقامی لغت (Cloud Native Glossary)"
+---
+
+
+# کلاؤڈ کی مقامی لغت
+
+Cloud Native Glossary ایک پروجیکٹ ہے جس کی قیادت CNCF بزنس ویلیو سب کمیٹی (BVS) کرتی ہے۔
+اس کا مقصد کسی بھی سابقہ ​​تکنیکی علم کی ضرورت کے بغیر کلاؤڈ کے مقامی تصورات کو واضح اور سادہ زبان میں بیان کرنا ہے۔
+
+<p><img class="mt-5" src="/images/homepage/stage.jpg" alt="A woman and two men presenting technical info on a stage"></p>
+
+## تعاون کرنا
+
+ہر ایک کو کلاؤڈ مقامی لغت میں تبدیلیوں، اضافے اور بہتری کی تجویز دینے کے لیے مدعو کیا جاتا ہے۔
+ہم اس مشترکہ لغت کو تیار کرنے اور بہتر بنانے کے لیے CNCF کے زیرانتظام کمیونٹی پر مبنی عمل کو استعمال کرتے ہیں۔
+یہ لغت ایک وینڈر غیر جانبدار پلیٹ فارم مہیا کرتی ہے تاکہ کلاؤڈ کی مقامی ٹیکنالوجیز کے ارد گرد مشترکہ الفاظ کو منظم کیا جا سکے۔
+منصوبے کے مقصد اور چارٹر کی پابندی کرنے والے تمام شرکاء کی طرف سے تعاون کا خیرمقدم کیا جاتا ہے۔
+
+کوئی بھی جو شراکت کرنا چاہتا ہے وہ GitHub کا مسئلہ جمع کرا سکتا ہے یا پل کی درخواست بنا سکتا ہے۔
+براہ کرم اس بات کو یقینی بنائیں کہ آپ [اسٹائل گائیڈ](/style-guide/)کی پیروی کرتے ہیں، دستاویز  [کیسے تعاون کریں ](/contribute/)کو پڑھیں، اور  CNCF Slack پر[glossary#](https://cloud-native.slack.com/archives/C02TX20MQBB) چینل میں شامل ہوں۔
+ان لوگوں کے لیے ایک [glossary-localizations#](https://cloud-native.slack.com/archives/C02N2RGFXDF) چینل بھی ہے جو اپنی مادری زبان میں لغت کا ترجمہ کرنے میں مدد کرنا چاہتے ہیں۔
+
+## اعترافات
+
+کلاؤڈ مقامی لغت CNCF مارکیٹنگ کمیٹی (بزنس ویلیو سب کمیٹی) کے ذریعہ شروع کی گئی تھی اور اس میں ان کے تعاون شامل ہیں
+[Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/)، 
+[Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/)، 
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk)، 
+[Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/)، 
+[Katelin Ramer](https://www.linkedin.com/in/katelinramer/)، 
+[Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca)، 
+اور بہت سے تعاون کنندگان۔ 
+شراکت داروں کی مکمل فہرست کے لیے، براہ کرم  [اس GitHub صفحہ](https://github.com/cncf/glossary/graphs/contributors) کو دیکھیں۔
+
+لغت کی دیکھ بھال 
+[Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/)، 
+[Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/)، 
+[Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/)، 
+[Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/) 
+، اور  [Seokho Son](https://www.linkedin.com/in/seokho-son/) نے کی۔
+
```

Comment:
Hi @waleed318 @Saim-Safdar 

You can add ack for localization contributors. :)

for instance, 
(please note that the example is just a machine translation of Korean version)

```suggestion

The Korean localization of the Cloud Native Terminology was contributed by [Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/) , [Seokho Son](https://www.linkedin.com/in/seokho-son/) , [Yunkon Kim](https://www.linkedin.com/in/%EC%9C%A4%EA%B3%A4-%EA%B9%80-47257914a/) and various contributors , including the Cloud Native Terminology Koreanization Team ([#glossary-localization-korean](https://cloud-native.slack.com/archives/C02PC0MLQKU)). 

```

Reply:
I'll take a look

Reply:
Thanks for pointing this out, let us add in the required info.